README.md

```diff
@@ -64,7 +70,7 @@ mvn compile exec:java -Dexec.mainClass="com.sforce.cd.apexUnit.ApexUnitRunner"
 *Please replace all $xyz with the values specific to your environment/project*
 
 Required parameters: 
-- -org.login.url : Login URL for the org
+- -org.login.url : Login URL for the org (for example, https://na14.salesforce.com)
```

Comment:
Do we need to have this specific? Won't login.salesforce.com be the good like test.salesforce.com and then they will get redirected to NA14 or CS46 let's say as needed. I don't know if everyone is familiar with the specific server they get redirected to?